src/main/java/org/opentripplanner/transit/service/StopModel.java

```diff
@@ -23,6 +23,8 @@
 
 /**
  * Repository for Stop entities.
+ *
```

Comment:
I think this was pasted here by mistake, isn't it?

Reply:
Yes